app.js

```diff
@@ -0,0 +1,23 @@
+const body = document.querySelector("body");
```

Comment:
Is this variable actually used?

Reply:
No - it was actually removed when I refactored so will check it is updated. Thanks!